translate/storage/csvl10n.py

```diff
@@ -25,72 +25,9 @@
 import six
 
 from translate.misc import csv_utils
-from translate.misc import sparse
```

Comment:
Good point, I've looked only at SimpleParser class which is used in selector, but it's also defined there...